flux-pipe/pdl_pipe_helper.pm

```diff
@@ -0,0 +1,193 @@
+    }
+    return $highest_numbered_file ? "$directory$highest_numbered_file" : 0, $highest_number;
+}
+
+sub set_env_variable {
+    my ($variable, $value) = @_;
+    $ENV{$variable} = $value;
+    return $ENV{$variable};
+}
+
+sub get_env_variable {
+    my ($variable) = @_;
+    return $ENV{$variable};
+}
+
+sub check_env_variable {
+    my ($variable, $print) = @_;
+    my $value = $ENV{$variable};
+    if (defined $value) {
+        if (defined $print) {
+            if ($print) {
+            print "\$$variable: \t$value\n";}}
+        return $value;
+    } else {
+        print "\$$variable is not set\n";
+        exit();
+    }
+}
+
+sub set_and_check_env_variable {
+    my ($variable, $value, $print) = @_;
+    set_env_variable($variable, $value);
+    return check_env_variable($variable, $print);
+    # return $value;
+}
+
+sub calculate_directories {
+(my $basedir, my $batch_name, my $print) = @_;
+
+    # Calculate Directory Structure
+    my $fluxdir =  "$basedir/fluxon-mhd";
+        my $pipedir =  "$fluxdir/flux-pipe";
+        my $pdldir =   "$fluxdir/pdl/PDL";
+
+    my $datdir =   "$basedir/fluxon-data";
+        my $magdir =   "$datdir/magnetograms";
+        my $batchdir = "$datdir/batches/$batch_name";
+            my $logfile = "$batchdir/pipe_log.txt";
+    
+    set_and_check_env_variable('FLUXPATH', $fluxdir, 0);
+    set_and_check_env_variable('PIPEPATH', $pipedir, $print);
+    set_and_check_env_variable('BATCHPATH', $batchdir, $print);
+    # print "\n";
+    set_and_check_env_variable('DATAPATH', $datdir, $print);
+    return ($pipedir, $pdldir, $datdir, $magdir, $batchdir, $logfile);
+}
+
+sub set_python_path {
+    my ($pythonpath, $print) = @_;
+    set_and_check_env_variable('PYTHONPATH', $pythonpath, $print);
+    return $pythonpath;
+}
+
+sub print_banner {
+    my ($batch_name, $CR, $reduction, $n_fluxons_wanted, $recompute_string) = @_;
+    print "|\n|\n|\n|\n|\n|\n|\n|\n|\n|";
+    print "\n\n";
+    print "--------------------------------------------------------------------------------------------------\n";
+    print "FLUXPipe: Indicate a Carrington Rotation and this script will run the entire Flux Pipeline for it.\n";
+    print "--------------------------------------------------------------------------------------------------\n";
+    print "\n\n";
+
+    # check_env_variable('PYTHONPATH', 1);
+    check_env_variable('DATAPATH', 1);
+    print "\nBatch: $batch_name, CR: $CR, Reduction: $reduction, Fluxons: $n_fluxons_wanted";
+
+    my $time = localtime;
+    my $ftime = $time->strftime('%m-%d-%Y %H:%M:%S');
+
+    print"\n\n";
+    print "\t>>>>>>>>>>>>>>>>>>>>> Recompute = $recompute_string <<<<<<<<<<<<<<<<<<<<<<";
+    print "\n\tStarting FLUXPipe at $ftime ";
+    print "\n\t>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>><<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<\n";
+    print "\n\n";
+    print "--------------------------------------------------------------------------------------------------\n";
+
+}
+
+
+use strict;
```

Comment:
Does moving these two imports to live with the others at the top of the file break anything?